Public/Src/Cache/ContentStore/Library/Stores/MemoryContentDirectory.cs

```diff
@@ -620,13 +620,11 @@ public Task SyncAsync()
         /// <summary>
         ///     Transform entries of an existing serialized file.
         /// </summary>
-        public static async Task TransformFile
-            (
+        public static async Task TransformFile(
             Context context,
             IAbsFileSystem fileSystem,
             AbsolutePath directoryPath,
-            Func<KeyValuePair<ContentHash, ContentFileInfo>, KeyValuePair<ContentHash, ContentFileInfo>> transformer
-            )
+            Func<KeyValuePair<ContentHash, ContentFileInfo>, KeyValuePair<ContentHash, ContentFileInfo>> transformer)

```

Comment:
Your new formatting is preferred and more consistent, but this change is unrelated to feature and adds a whole new file to PR.

Reply:
will revert.

---
In reply to: [283070929](https://github.com/microsoft/BuildXL/pull/289#discussion_r283070929) [](ancestors = 283070929)